python/cudf/cudf/tests/test_struct.py

```diff
@@ -392,3 +392,13 @@ def test_struct_with_null_memory_usage():
 
     s[2:4] = None
     assert s.memory_usage() == 272
+
+
+@pytest.mark.parametrize("indices", [slice(0, 3), slice(1, 4)])
+def test_struct_empty_children_nulls_slice(indices):
+    values = [None, {}, {}, None]
+
+    s = cudf.Series([None, {}, {}, None])
```

Comment:
Reuse the input from the previous line if it’s equivalent.
```suggestion
    s = cudf.Series(values)
```